tools/scripts/data.toml

```diff
@@ -48,21 +60,6 @@ args = [
     "--overwrite",
 ]
 
-[tasks.testdata-build-hello-world]
```

Comment:
Issue: This is used to re-generate hello_world.postcard.

Please restore this task, and optionally add more comments or documentation either here or in a new file like provider/blob/tests/README.md.

Reply:
Ah this was showing as removed in #1460 because it hasn't been synced in a while.